app/index.html

```diff
@@ -25,6 +25,10 @@
   <script src="https://apis.google.com/js/platform.js" async defer></script>
   <!-- Angular Cookies -->
   <script src="https://ajax.googleapis.com/ajax/libs/angularjs/1.6.4/angular-cookies.js"></script>
+  <!-- Ng-file-upload -->
+  <!-- shim is needed to support non-HTML5 FormData browsers (IE8-9)-->
+  <script src="ng-file-upload-shim.min.js"></script>
+  <script src="ng-file-upload.min.js"></script>
```

Comment:
I'm pretty sure this will only work if you've downloaded the files locally, which is why you're getting the errors.